pilot/pkg/config/kube/ingressv1/controller.go

```diff
@@ -237,6 +238,17 @@ func (c *controller) RegisterEventHandler(kind config.GroupVersionKind, f func(c
 	}
 }
 
+func (c *controller) SetWatchErrorHandler(handler func(r *cache.Reflector, err error)) error {
+	var errs error
+	if err := c.serviceInformer.SetWatchErrorHandler(handler); err != nil {
```

Comment:
So this serviceInformer is actually shared by other places - does this handle it ok? IIRC its "last one wins" which is fine if they always use the same handler?

Reply:
ah, I was hoping that the metric included a label indicating which component was using the informer, that part won't be accurate – the more important part is the cluster label

I'll trim this down a bit